torch_xla/csrc/runtime/runtime.cpp

```diff
@@ -1,50 +1,79 @@
 #include <torch/csrc/lazy/backend/backend_device.h>
 
+#include "absl/log/absl_check.h"
 #include "torch_xla/csrc/device.h"
 #include "torch_xla/csrc/runtime/computation_client.h"
 #include "torch_xla/csrc/runtime/env_vars.h"
 #include "torch_xla/csrc/runtime/ifrt_computation_client.h"
 #include "torch_xla/csrc/runtime/pjrt_computation_client.h"
 #include "tsl/platform/stacktrace_handler.h"
 
-namespace torch_xla {
-namespace runtime {
+namespace torch_xla::runtime {
 
 std::atomic<bool> g_computation_client_initialized(false);
 
-ComputationClient* GetComputationClient() {
-  static std::unique_ptr<ComputationClient> client = []() {
-    if (sys_util::GetEnvBool("XLA_DUMP_FATAL_STACK", false)) {
-      tsl::testing::InstallStacktraceHandler();
-    }
+namespace status {
+
+static absl::StatusOr<std::unique_ptr<ComputationClient>> InitializeComputationClient() {
```

Comment:
Document this? In particular, use `absl_nonnull` to indicate that the returned unique_ptr cannot be null.